ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -146,6 +153,17 @@ private[testtool] final class ParticipantTestContext private[participant] (
   def participantId(): Future[String] =
     services.partyManagement.getParticipantId(new GetParticipantIdRequest).map(_.participantId)
 
+  def listPackages(): Future[Seq[String]] =
+    services.packages.listPackages(new ListPackagesRequest(ledgerId)).map(_.packageIds)
+
+  def getPackage(packageId: String): Future[GetPackageResponse] =
```

Comment:
Would it make sense to simply return the byte array directly here?

Reply:
`GetPackageResponse` also contains the hash value and the hash type (although the hash value *should* be the same as the input `packageId`).